packages/ERTP/src/types.js

```diff
@@ -51,51 +51,22 @@
  * @param {Value} allegedValue
  * @returns {Amount}
  *
- * TODO find out how to get this "deprecated" marking recognized,
- * or remove it.
- * @deprecated Use brand-first overload instead
- * @callback AmountMakeValueBrand
- * Please use the brand-first overload. The value-first overload
- * is deprecated and will go way.
- * @param {Value} brand
- * @param {Brand} allegedValue
- * @returns {Amount}
- *
- * @typedef {AmountMakeBrandValue & AmountMakeValueBrand} AmountMake
+ * @typedef {AmountMakeBrandValue} AmountMake
```

Comment:
IIUC, this makes `AmountMakeBrandValue` and `AmountMake` synonymous. Why not just pick one and use it consistently? Prefer the shorter name unless there's a reason to avoid it.

This issue occurs throughout this file below.

Reply:
Ah, I missed this indirection that was added for the deprecated version when I removed the deprecated version.